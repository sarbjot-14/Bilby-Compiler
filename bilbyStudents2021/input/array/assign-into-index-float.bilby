main{
	
    
    imm f := [11.4,12.2,13.7,14.11,15.99];
	
	f[0] :=  0.0;
	
	f[1] :=  22.22;
	
	f[4] :=  555.55;
	
    print f $n;
		

}