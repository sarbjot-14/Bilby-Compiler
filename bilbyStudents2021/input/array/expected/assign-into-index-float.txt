[0.000000, 22.220000, 13.700000, 14.110000, 555.550000]
